README.md

```diff
@@ -97,6 +97,43 @@ get('/oauth2callback') do
 end
 ```
 
+### Example (Web with PKCE)
+
+```ruby
+require 'googleauth'
+require 'googleauth/web_user_authorizer'
+require 'googleauth/stores/redis_token_store'
+require 'redis'
+
+client_id = Google::Auth::ClientId.from_file('/path/to/client_secrets.json')
+scope = ['https://www.googleapis.com/auth/drive']
+token_store = Google::Auth::Stores::RedisTokenStore.new(redis: Redis.new)
+authorizer = Google::Auth::WebUserAuthorizer.new(
+  client_id, scope, token_store, '/oauth2callback')
+
+
+get('/authorize') do
+  # NOTE: Assumes the user is already authenticated to the app
+  user_id = request.session['user_id']
+  # User needs to take care of generating the code_verifier and storing it in
+  # the session.
+  request.session['code_verifier'] ||= authorizer.generate_code_verifier
+  authorizer.code_verifier = request.session['code_verifier']
```

Comment:
Is this how a user can generate the code verifier? It may be useful to give a short overview on what PKCE is

Reply:
Yes. Cool, added an overview.